origin_ca.go

```diff
@@ -0,0 +1,147 @@
+package cloudflare
+
+import (
+	"encoding/json"
+
+	"github.com/pkg/errors"
+)
+
+// OriginCA resource
+// This is the origin_ca resource definition (as defined here:
+// https://api.cloudflare.com/#origin-ca || https://api.cloudflare.com/#cloudflare-ca
+type OriginCA struct {
+	ID              string   `json:"id"`
+	Certificate     string   `json:"certificate"`
+	Hostnames       []string `json:"hostnames"`
+	ExpiresOn       string   `json:"expires_on"`
```

Comment:
I think it makes better sense to return dates as `time.Time` types, I have made #68 to change it for the rest of the code.